Defs/RecipeDefs/Recipes_Production.xml

```diff
@@ -70,5 +70,43 @@
 		</products>
 	</RecipeDef>
 
+	<RecipeDef>
+		<defName>ReclaimMechAmmo</defName>
```

Comment:
Honestly, I might include a bunch of different varieties of charge ammo. Additionally, you could add a more efficient version for the fabrication bench.